airbyte-commons-server/src/main/java/io/airbyte/commons/server/handlers/OAuthHandler.java

```diff
@@ -294,14 +289,14 @@ public void setDestinationInstancewideOauthParams(final SetInstancewideDestinati
   private JsonNode getOAuthInputConfigurationForConsent(final ConnectorSpecification spec,
                                                         final JsonNode hydratedSourceConnectionConfiguration,
                                                         final JsonNode oAuthInputConfiguration) {
-    final Map<String, String> fieldsToGet =
-        buildJsonPathFromOAuthFlowInitParameters(OAuthPathExtractor.extractOauthConfigurationPaths(
+    final Map<String, String> fieldsToGet = buildJsonPathFromOAuthFlowInitParameters(
```

Comment:
there are formatting changes in this file that suggest to me you have a different line width/format set up in your Intellij. It should be set to 150 and using the style guide as noted in our public docs. Can you check that?

Reply:
somewhat distracting from the main changeset, which is lines 364 onwards

Reply:
Yeah apologies, I thought I had that set up, will check it now

Reply:
You're definitely right, I missed that step in setup. Just updated it.

Reply:
reformatted things as well to match the original as closely as possible with 1 or 2 exceptions where I thought it made sense.